synapseclient/table.py

```diff
@@ -1242,7 +1242,7 @@ def Table(schema, values, **kwargs):
     try:
         import pandas as pd
         pandas_available = True
-    except:
+    except:  # noqa
```

Comment:
except ImportError

Reply:
There is another [import pandas ](https://github.com/Sage-Bionetworks/synapsePythonClient/blob/67db15cf5f32fcb4e7953396df192712f82ff733/synapseclient/table.py#L365) in this file that implies that an import can somehow raise other errors, the associated [bug](https://sagebionetworks.jira.com/browse/SYNPY-177) implies an AttributeError is possible, which seems strange, but given that a previous bug exists for this I was not confident in narrowing the Exception.